community/examples/hpc-slurm-chromedesktop.yaml

```diff
@@ -74,8 +78,8 @@ deployment_groups:
   - id: compute_node_group
     source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
     settings:
-      machine_type: n2-standard-4
-      node_count_dynamic_max: 1
+      machine_type: n2d-standard-16
+      node_count_dynamic_max: 20
```

Comment:
Just for my understanding, why do we want to set this to large number ?

Reply:
This is the max number of nodes that can be dynamically scaled to. If set to one then we can only create 1 node which is not much of a cluster. In a real world situation we would probably have a need to run multiple nodes. Since this is a max and nodes are dynamic, it will only deploy the nodes if there are jobs waiting for them.